tests/suite-unit/PluginFlyvemdmTaskstatus.php

```diff
@@ -58,4 +58,29 @@ public function testGetTypeName() {
       $instance = $this->createInstance();
       $this->string($instance->getTypeName())->isEqualTo('Task status');
    }
+
+   /**
+    * @tags testUpdateStatus
+    */
+   public function testUpdateStatus() {
+    $testPolicy = new PluginFlyvemdmPolicyBase();
+    $instance = $this->createInstance();
+    list($policy) = $testPolicy->createNewPolicyInstance();
+        $status = $policy->filterStatus("pending");
+        $this->notNull($status);
+        $instance->update([
+           'id'     => $instance->getID(),
+           'status' => $status,
+        ]);
+        $this->string($status)->isEqualTo($instance->getField('status'));
+   }
```

Comment:
Indentation problem ?

Reply:
Hi, @flyve-mdm/backend-development 
@Gianfranco97 is using `.editorconfig` in the Web Dashboard repo.
Reference:
https://editorconfig.org

Reply:
ok, i will it.

Reply:
@ingluife have a look on this: https://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html